### ISSUE TITLE
How to install superset in the Linux environment without network?

### ISSUE THREAD

**Issue by weylin0714** [apache/incubator-superset#7871]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by syazwan0913** [comment]
Superset require dependencies as in requirements.txt, so network is needed for the first time as dependencies need to be downloaded.

**Comment by atifahsan** [comment]
I did it on RHEL7 without the internet connection it was a very tedious
task. What I did was create a similar distribution machine on AWS and
downloaded all system dependencies using yum on it and then copy and
install them on the offline machine. Similarly pip download all packages
from requirements.txt and then copy and install them on offline machine.

Regards
Atif Ahsan
+971 50 728 6001

**Comment by dpgaspar** [comment]
On AWS you can always "bake" an AMI with all the dependencies, there are several other methods one can be to pre package all your pip requirements using: https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-t

**Comment by jonaycp** [comment]
I did the installation on a server without internet manually. 

- First make sure that OS dependencies are satisfied. 
Some (not sure if all) dependencies could be:
gcc, gcc-c++, libffi-devel, python-devel, python-pip, python-wheel, openssl-devel, libsasl2-devel, openldap-devel

- Then I downloaded on an OS with internet all the packages on the requirements.txt
Example:
`pip download <Lib. name>==<version>
(If you leave <version> empty you will see the available versions for that package)

- Next, I put everything in the same folder and I copied on the server without internet, and then I installed superset with:
`pip install superset --no-index --find-links=/path-to-packages/`
In my case for some reason the installation stopped installing some libs, in that case I just tried with other versions until it works
